src/app/cli/src/commands/ingest_command.rs

```diff
@@ -30,13 +31,14 @@ pub struct IngestCommand {
     dataset_ref: DatasetRef,
     files_refs: Vec<String>,
     source_name: Option<String>,
+    event_time: Option<String>,
     stdin: bool,
     recursive: bool,
     input_format: Option<String>,
 }
 
 impl IngestCommand {
-    pub fn new<'s, I, S1, S2>(
+    pub fn new<'s, I, S1, S2, S3>(
```

Comment:
Just a proposal, could we use other namings for generics? Once I saw s3 first my thought was it is something connected with s3 bucket

Reply:
We propably should get rid of this pattern altogether and just accept `&str`